### ISSUE TITLE
How to specify the output file format of "Sequences of files"

### ISSUE THREAD

**Issue by ZenanH** [w1th0utnam3/splashsurf#174]
Hi! I'm using splashsurf in my project, it is very great!👍

In my case, I need the surface format in `.obj` instead of the default `.vtk`. I can specify the output format by `-o filename.obj` for single file, however it cannot work when I use the  "sequences of files", here is my code: (it still generates files in `.vtk` format)
```
splashsurf reconstruct -s "model_{}.ply" --output-dir=out -o "test_{}.obj" --mt-files=on --mt-particles=off --particle-radius=0.00187 --smoothing-length=1.2 --cube-size=0.5 --surface-threshold=0.6
```

<br>

For now, I wrote a for-loop in `.bat` file, it worked with a bad efficiency. So, I'm wondering how to generate `.obj` directly?

**Comment by w1th0utnam3** [comment]
I checked the code and at the moment the sequence processing feature only supports outputting VTK files. Next week I can look into updating the code to support all of the output formats.

Usually the batch script should not be slower than the sequence processing in splashsurf, except if you have only very few particles which would make it faster to process multiple files in parallel instead of processing particles inside of one file in parallel.
If this is the case, for now you could consider using Python to start multiple processes in parallel using `multiprocessing`: https://stackoverflow.com/questions/46542957/how-to-run-an-external-program-in-parallel-mode-using-python3
However, in this case you should disable parallel processing for splashsurf itself with the command line option `--mt-particles=off`

**Comment by w1th0utnam3** [comment]
@ZenanH I just published a new release (v0.9.0) which allows you to directly output .obj files from a sequence. If you use `--input-sequence=file_{}.vtk`, just specify additionally e.g. `--output-file=file_surface_{}.obj` and it should work. 

You can install the latest version of splashsurf normally using `cargo install splashsurf`.

**Comment by w1th0utnam3** [comment]
Thanks for testing this. There appears to be a bug in the code that tries to find the files belonging to the sequence if there is no path in front of the filename. I will fix it.

For now it should work if you specify either the full path or a relative path in front of the filename, like `-s ".\model_{}.ply"`.

**Comment by w1th0utnam3** [comment]
@ZenanH I just published version 0.9.1 which should fix this problem.
